client/app/views/popover_screens/duplicate.coffee

```diff
@@ -0,0 +1,38 @@
+PopoverScreenView = require 'lib/popover_screen_view'
+
+module.exports = class ConfirmDuplicatePopoverScreen extends PopoverScreenView
```

Comment:
From file name/path, I feel that the class name should be `DuplicatePopoverScreen`. Am I right ?